fog/overseer/server/src/worker.rs

```diff
@@ -235,12 +246,23 @@ where
     /// Returns the latest round of ingest summaries for each
     /// FogIngestGrpcClient that communicates with a node that is online.
     fn retrieve_ingest_summary_node_mappings(
-        &self,
+        &mut self,
     ) -> Result<Vec<IngestSummaryNodeMapping>, OverseerError> {
         let mut ingest_summary_node_mappings: Vec<IngestSummaryNodeMapping> = Vec::new();
         for (ingest_client_index, ingest_client) in self.ingest_clients.iter().enumerate() {
             match ingest_client.get_status() {
                 Ok(ingest_summary) => {
+                    if self
+                        .unresponsive_node_urls
+                        .contains(ingest_client.get_uri())
+                    {
+                        self.unresponsive_node_urls.remove(ingest_client.get_uri());
+                        log::info!(
+                            self.logger,
+                            "Node {} was previously unresponsive, but just successfully responded!",
```

Comment:
Would it make sense to decrement `unresponsive_node_count` here?

Reply:
I think we want to keep a running total of how many unresponsive nodes there have been. If we wanted a snapshot of unresponsive node count, then we'd use a prometheus gauge.